### PR TITLE
Implement Pull #3027 from citra: travis: Add build with frozen versions for dependencies 

### LINKED ISSUE #40: Things that should be ported over from Citra
[This](https://pastebin.com/ECBKedRb) is a list @jroweboy posted, for features that would be good to be ported over from Citra. The checked ones are already implemented.

- [ ] [#3027](https://github.com/citra-emu/citra/pull/3027)  (In review)
- [x] [#3001](https://github.com/citra-emu/citra/pull/3001)
- [x] [#3116](https://github.com/citra-emu/citra/pull/3116)
- [x] [#3333](https://github.com/citra-emu/citra/pull/3333)
- [x] [#3005](https://github.com/citra-emu/citra/pull/3005)
- [x] [#2972](https://github.com/citra-emu/citra/pull/2972)
- [x] [#3240](https://github.com/citra-emu/citra/pull/3240)
- [x] [#3052](https://github.com/citra-emu/citra/pull/3052)
- [x] [#3034](https://github.com/citra-emu/citra/pull/3034)
- [x] [#3030](https://github.com/citra-emu/citra/pull/3030)
- [x] [#3318](https://github.com/citra-emu/citra/pull/3318)
- [x] [#3109](https://github.com/citra-emu/citra/pull/3109)
- [x] [#2949](https://github.com/citra-emu/citra/pull/2949)
- [x] [#3295](https://github.com/citra-emu/citra/pull/3295)
- [x] [#3306](https://github.com/citra-emu/citra/pull/3306)
- [x] [#3093](https://github.com/citra-emu/citra/pull/3093)
- [x] [#3336](https://github.com/citra-emu/citra/pull/3336)
- [x] [#2912](https://github.com/citra-emu/citra/pull/2912)
- [x] [#2918](https://github.com/citra-emu/citra/pull/2918)
- [x] [#2982](https://github.com/citra-emu/citra/pull/2982)
- [x] [#2849](https://github.com/citra-emu/citra/pull/2849)
- [x] [#3320](https://github.com/citra-emu/citra/pull/3320)
- [x] [#3352](https://github.com/citra-emu/citra/pull/3352)
- [x] [#3275](https://github.com/citra-emu/citra/pull/3275)
- [x] [#3064](https://github.com/citra-emu/citra/pull/3064)
- [x] [#3043](https://github.com/citra-emu/citra/pull/3043)
- [x] [#3184](https://github.com/citra-emu/citra/pull/3184)
- [x] [#3101](https://github.com/citra-emu/citra/pull/3101) 

PRs by lioncash which he will update himself
- [x] [#3161](https://github.com/citra-emu/citra/pull/3161)
- [x] [#3262](https://github.com/citra-emu/citra/pull/3262)
- [x] [#3263](https://github.com/citra-emu/citra/pull/3263)
- [x] [#3264](https://github.com/citra-emu/citra/pull/3264)
- [x] [#3266](https://github.com/citra-emu/citra/pull/3266)
- [x] [#3294](https://github.com/citra-emu/citra/pull/3294)
- [x] [#3312](https://github.com/citra-emu/citra/pull/3312)

Feel free to take over one of the mentioned PRs. Just leave a comment and I will mention it in the list. I will try to keep the list up to date while we are making more progress.

### PR DESCRIPTION
**Pull request by N00byKing**
Continuation of #126 

Closes #40

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,6 +25,16 @@ matrix:
       install: "./.travis/linux/deps.sh"
       script: "./.travis/linux/build.sh"
       after_success: "./.travis/linux/upload.sh"
+    - os: linux
+      env: NAME="linux build (frozen versions of dependencies)"
+      sudo: required
+      dist: trusty
+      services: docker
+      before_install:
+        - chmod +x ./.travis/linux-frozen/build.sh
```
Comment (jroweboy) [inline .travis.yml]: is there any reason these files are not already executable in the repo?

Comment (N00byKing) [reply .travis.yml]: Will make them executable when I get home

**Comment by N00byKing** [comment]
@jroweboy all done!

**Comment by MerryMage** [comment]
This feels like this should perhaps be something in the future when yuzu is more stable?

**Comment by N00byKing** [comment]
Dunno, i can close it if you want tho

**Comment by Hexagon12** [comment]
What's the status of this PR?

**Comment by N00byKing** [comment]
@Hexagon12 theoretically finished. Whether or not it should be merged seems unclear though

**Comment by N00byKing** [comment]
Its been a while, but I rebased it now, awoken from my slumber by Email-Notification of #40's closing

**Comment by Hexagon12** [comment]
If the PR is good enough, please merge, since it has been sitting for ages + doesn't have conflicts

**Pull request closed without merging**